src/utils/color.js

```diff
@@ -61,3 +61,43 @@ export function importFonts(fontNames) {
 		document.head.appendChild(link);
 	});
 }
+
+export function importFonts(fontNames) {
```

Comment:
Why did you keep the old function?

Reply:
I just modified the old function to add a few additional checks.  Should I have replaced it with something else entirely?

Reply:
the old function is still here, a bit higher =)